docs/rfcs/xxxx-backwards-compatibility.md

```diff
@@ -0,0 +1,176 @@
+  1.0 release.
+
+- **Telemetry data used in official dashboards**. This means that users will
+  continue to be able to use the same set of dashboards we provide when
+  upgrading minor releases.    
+
+- **Externally importable Go packages**. If a user is importing our code as a
+  dependency, they should be able to upgrade to a new minor release without
+  having to make changes to their code.
+
+  This means that names, signatures, and meanings may not change; a function
+  called Calculate may not be changed from returning the sum of two numbers to
+  returning the difference of two numbers.
+
+  Protecting Go packaged with backwards compatibility is expected to be a
+  contentious proposal. However, it has positive benefits: 
+
+  - It makes us good open-source citizens, allowing people to use Grafana Agent
+    as a library with the same upgrade confidence as people who use Grafana
+    Agent as a binary.  
+
+  - It allows us to get in the habit of being deliberate around when something
+    should be consumable downstream, and will help us strengthen our skills
+    with designing good APIs in general. Ideally, this helps us get better at
+    writing Go code _and_ writing Flow components.
+
+  - It will help train us to more easily recognize when any kind of change is
+    breaking. 
+
+  The easiest way to commit to this without any effort is to put everything in
+  an top-level internal folder, effectively saying that Grafana Agent has no
+  code API that can be consumed downstream.
+
+- **The scope of backwards compatibility**. Backwards compatibility is only
+  defined for major version 1; we reserve the right to change the definition of
+  backwards compatibility between major versions. 
+
+If a breaking change is introduced in a minor change accidentally, and that
+breaking change is not covered by one of the [exceptions][] defined below, it
+is a bug. In these cases, a patch release should be introduced to undo the
+breaking change. 
+
+[exceptions]: #exceptions-to-backwards-compatibility
+
+### Exceptions to backwards compatibility 
+
+It's impossible to guarantee that full backwards compatibility is achieved.
+There are some exceptions which may cause a breaking change without a new major
+version:
+
+- Security: a breaking change may be made if a security fix requires making a
+  breaking change. 
+
+- Non-versioned network APIs: Internal network APIs, such as the internal API
+  used to drive the Flow web UI, are not subject to backwards compatibility
+  guarantees.
+
+- Undocumented behavior: relying on undocumented behavior may break between
+  minor releases. 
+
+- Upstream dependencies: part of the public API of Grafana Agent may directly
+  expose the public API of an upstream dependency. In these cases, if an
+  upstream dependency introduces a breaking change, we may be required to make
+  a breaking change to our public API as well.   
+
+- Other telemetry data: Metrics, logs, and traces may change between releases.
```

Comment:
```suggestion
- Other telemetry data: metrics, logs, and traces may change between releases.
```

Reply:
Not sure if you intend to have a capital letter after the colon or the other way around.